src/items/__tests__/item.service.spec.ts

```diff
@@ -224,4 +228,33 @@ describe('ItemService', () => {
       });
     });
   });
+
+  describe('When delistAnItem function is called ', () => {
```

Comment:
What an amazing test.. Where are you learning this crap? :P

Reply:
O.o